<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplePlugin</class>
 <widget class="QDockWidget" name="SamplePlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Original Plugin</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>450</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="_btn0">
           <property name="text">
            <string>Load Marker</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btn1">
           <property name="text">
            <string>See Robot View</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Robotics Test</string>
       </attribute>
       <widget class="QPushButton" name="_btn_rotest_loadMarker">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>231</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Marker Movement</string>
        </property>
       </widget>
       <widget class="QPushButton" name="_btn_rotest_computeQP1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Compute Q for 1 point</string>
        </property>
       </widget>
       <widget class="QSlider" name="_slider_rotest_Q">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>411</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="_btn_rotest_computeQP3">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>70</y>
          <width>201</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Compute Q for 3 points</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>ROVI</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>19</y>
          <width>411</width>
          <height>128</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="deviceNameLabel">
           <property name="text">
            <string>Device Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="_line_settings_devName">
           <property name="text">
            <string>PA10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="tCPLabel">
           <property name="text">
            <string>TCP</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="_line_settings_tcp">
           <property name="text">
            <string>CameraSim</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="markerLabel">
           <property name="text">
            <string>Marker</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="_line_settings_marker">
           <property name="text">
            <string>Marker</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="timestepLabel">
           <property name="text">
            <string>Timestep</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="_spinBox_timestep">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="_label">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
